templates/atom/apps/qubit/config/app.yml

```diff
@@ -4,11 +4,12 @@ all:
 
   upload_limit: {{ atom_app_upload_limit }}
 
-  cache_engine: {{ atom_app_cache_engine }}
+  cache_engine:
```

Comment:
The cache config should only be changed for older versions of AtoM and Binder/DRMC as it uses AtoM 2.1